hack/release-manifest.py

```diff
@@ -0,0 +1,96 @@
+import os
+import yaml
+import sys
+from pathlib import Path
+
+OPERATOR_VERSION = str(sys.argv[2])
+GIT_COMMIT_ID = str(sys.argv[3])
+path = os.getcwd()
+release_path = os.path.join(path, "out", "releases", OPERATOR_VERSION)
+os.makedirs(release_path)
+release_yaml = path + "/out/releases/" + OPERATOR_VERSION + "/release.yaml"
```

Comment:
pass the full path to `release.yaml` as the script arg.

Reply:
Any particular reason for that? I thought of passing min args to the script